gateway/src/apicast/cli/command/start.lua

```diff
@@ -75,40 +78,73 @@ local function get_log_level(self, options)
     return log_level
 end
 
-function mt:__call(options)
-    local openresty = resty_env.value('APICAST_OPENRESTY_BINARY') or
-                      resty_env.value('TEST_NGINX_BINARY') or
-                      pick_openesty(self.openresty)
-    local dir = resty_env.get('APICAST_DIR') or pl.path.abspath('.')
-    local config = configuration.new(dir)
-    local path = options.template
-    local environment = options.dev and 'development' or options.environment
-    local context = config:load(environment)
-    local env = {
+
+local function build_environment_config(options)
+    local config = Environment.new()
+
+    for i=1, #options.environment do
+        local ok, err = config:add(options.environment[i])
+
+        if not ok then
+            print('not loading ', options.environment[i], ': ', err)
+        end
+    end
+
+    if options.dev then config:add('development') end
+
+    config:save()
+
+    return config
+end
+
+local function openresty_binary(candidates)
+    return resty_env.value('APICAST_OPENRESTY_BINARY') or
+        resty_env.value('TEST_NGINX_BINARY') or
+        pick_openesty(candidates)
```

Comment:
typo, also, I find naming here a bit confusing. This method is choosing the openresty binary but it's calling one named `pick_openresty`.

Reply:
Yes. I was too lazy to rename this function. Will do 👍

Reply:
Fixed in https://github.com/3scale/apicast/pull/496/commits/d5a7ca002f60bca98e1dba734238ead55b583050 and will squash it.